README.md

```diff
@@ -1,164 +1,54 @@
 
-## Usage
+Canonical has developed an open-source operator called  Charmed MongoDB, making it easier to operate MongoDB. The Charmed MongoDB Virtual Machine (VM) operator deploys and operates MongoDB on physical,  Virtual Machines (VM) and other wide range of cloud and cloud-like environments, including AWS, Azure, OpenStack and VMWare.
 
-### Basic Usage
-To deploy a single unit of MongoDB using its default configuration
-```shell
-juju deploy mongodb --channel 5/edge
-```
+Charmed MongoDB (VM Operator) is an enhanced, open source and fully-compatible drop-in replacement for the MongoDB Community Edition with advanced MongoDB enterprise features. It simplifies the deployment, scaling, design and management of MongoDB in production in a reliable way. In addition, you can use the operator to manage your MongoDB clusters with automation capabilities. It also offers replication, TLS, password rotation, easy-to-use application integration, backup, restore, and monitoring.  
 
-It is customary to use MongoDB with replication. Hence usually more than one unit (preferably an odd number to prohibit a "split-brain" scenario) is deployed. To deploy MongoDB with multiple replicas, specify the number of desired units with the `-n` option.
-```shell
-juju deploy mongodb --channel 5/edge -n <number_of_replicas>
-```
+Aside from a VM operator, Canonical also developed another operator called [Charmed MongoDB (K8s operator)](https://charmhub.io/mongodb-k8s?channel=5/edge) that operates in the Kubernetes environment, and you can see the documentation [here](https://charmhub.io/mongodb-k8s?channel=5/edge).
 
-To retrieve primary replica one can use the action `get-primary` on any of the units running MongoDB
-```shell
-juju run-action mongodb/<unit_number> get-primary --wait
-```
 
-Similarly, the primary replica is displayed as a status message in `juju status`, however one should note that this hook gets called on regular time intervals and the primary may be outdated if the status hook has not been called recently. 
-
-Further we highly suggest configuring the status hook to run frequently. In addition to reporting the primary, secondaries, and other statuses, the status hook performs self healing in the case of a network cut. To change the frequency of the update status hook do:
-```shell
-juju model-config update-status-hook-interval=<time(s/m/h)>
-```
-Note that this hook executes a read query to MongoDB. On a production level server this should be configured to occur at a frequency that doesn't overload the server with read requests. Similarly the hook should not be configured at too quick of a frequency as this can delay other hooks from running. You can read more about status hooks [here](https://juju.is/docs/sdk/update-status-event).
+## Software and releases
 
```

Comment:
Charmed MongoDB (VM Operator) is an enhanced, open source and fully-compatible drop-in replacement for the MongoDB Community Edition with advanced MongoDB enterprise features. This operator uses the [Charmed MongoDB snap package](https://snapcraft.io/charmed-mongodb), which offers more features than the MongoDB Community version, such as backup and restores, monitoring and security features.

To see the Charmed MongoDB features and releases, visit our [Github Release](https://github.com/canonical/mongodb-operator/releases) and Features [Release Notes](https://discourse.charmhub.io/t/release-notes-charmed-mongodb-5-vm-operator/10039) page.